docs/source/configuration.rst

```diff
@@ -60,14 +60,14 @@ However:
 
 .. code-block:: shell
 
-   tartufo --config tartufo.toml --config myconfig.toml scan-local-repo .
+   tartufo --config tartufo.toml --config myconfig.toml scan-local-repo <target_directory>
 
-will cause ``tartufo`` to read ``tartufo.toml`` in the current directory
+will cause ``tartufo`` to read ``tartufo.toml`` in the target directory
 (assuming it exists, and in parent directories otherwise), and then ``myconfig.toml``
-as above, and then either ``tartufo.toml`` or ``pyproject.toml`` in the current
+as above, and then either ``tartufo.toml`` or ``pyproject.toml`` in the target
 directory (only) because that is the target of the scan.
 
-If ``tartufo.toml`` exists in the current directory, the effect is that
+If ``tartufo.toml`` exists in the target directory, the effect is that
 ``tartufo.toml`` is read first, ``myconfig.toml`` is read second (possibly
 overriding directives), and ``tartufo.toml`` is not read again because it was
```

Comment:
I think this documentation might no longer be accurate, because of the change above from `.` to `<target_directory>` .

Reply:
Agreed.

Reply:
The statement needed at the end is that tartufo.toml is not processed a second time if the file found in the target directory is the same file -- as measured by absolute path name -- as the tartufo.toml ultimately used by the first `--config` directive.